apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -297,12 +294,8 @@ NetSimLocalClientNode.prototype.connectToRouter = function (router, onComplete)
         return;
       }
 
-      this.myRouter = router;
```

Comment:
This was being set twice in this method; expand a few lines above to see the other set.